packages/rest/src/writer.ts

```diff
@@ -48,6 +50,14 @@ export function writeResultToResponse(
         result = result.toString();
         break;
     }
+    if (route) {
+      const spec = route.spec;
+      const responsesCode = Object.keys(spec.responses || {});
+      if (responsesCode.length >= 1) {
+        const successStatusCode = responsesCode[0];
```

Comment:
The assumption here is risky.

Reply:
We might need to allow `x-default-status-code`.

Reply:
What are the risks?

What if we filtered `responsesCode` to find the first "success" code in the range 2xx, would that mitigate the risks?